docs/sql/tutorials/mindsdb-superset-snowflake.mdx

```diff
@@ -174,8 +174,8 @@ The combination of MindsDB and your database covers all the phases of the ML lif
 
 
 MindsDB provides easy-to-use predictive models through AI Tables. You can create these predictive models using SQL statements and feeding the input data. Also, you can query them the same way you query a table. The easiest way to get started with Superset is with the free tier for [Preset Cloud](https://preset.io/product/), a hassle-free and fully hosted cloud service for Superset.
-
-We encourage you to try some predictions with your own data, so please sign up for a [free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask our [Slack](https://join.slack.com/t/mindsdbcommunity/shared_invite/zt-o8mrmx3l-5ai~5H66s6wlxFfBMVI6wQ) and [Github](https://github.com/mindsdb/mindsdb/discussions) communities.
+[Slack](https://mindsdb.com/joincommunity)
+We encourage you to try some predictions with your own data, so please sign up for a [free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask our  and [Github](https://github.com/mindsdb/mindsdb/discussions) communities.
```

Comment:
You moved the link to the previous line, which is not as in the instructions.

Please bring the link back to the sentence. The sentence should be as below:

We encourage you to try some predictions with your own data, so please sign up for a \[free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask **our \[Slack](https://mindsdb.com/joincommunity)** and \[Github](https://github.com/mindsdb/mindsdb/discussions) communities.